spec/controllers/api/v1/channels_controller_spec.rb

```diff
@@ -0,0 +1,194 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::ChannelsController, type: :controller do
+  let!(:organization) { create(:organization) }
+  let!(:channel) { create(:channel, organization: organization) }
+
+  context 'when token is valid' do
+    before do
+      request.headers['Token'] = organization.token
+    end
+
+    describe 'GET #show' do
+      it 'return channel' do
+        get :show, params: { id: channel.id }
+
+        json_response = JSON.parse(response.body)
+
+        expect(response).to have_http_status(200)
+        expect(json_response['id']).to eq(channel.id)
```

Comment:
it's even better to verify all data from the response, to ensure there's no some presenter/template mistake